packit_service/constants.py

```diff
@@ -256,3 +256,9 @@ def from_number(number: int):
     f"You were denied from using Packit Service. If you think this was done by mistake, "
     f"please, [let us know]({CONTACTS_URL})."
 )
+
+# We want to be able to access both
+# upstream and downstream repos trough the
+# shared sandcastle dir
+SANDCASTLE_DG_REPO_DIR = "dg"
+SANDCASTLE_LOCAL_PROJECT_DIR = "local-project"
```

Comment:
I know the names don't matter much, but if local project working dir is in fact the upstream repo, why not use `upstream` and `downstream`/`dist-git`?

Reply:
The `LocalProjectMixin` sets the suffix for this dir. The local project can be both an upstream or downstream project, it depends on the others mixins in the handler...
I can be sure only about the `dg` suffix, because it is used by the `PackitAPIWithUpstreamMixin` which is setting it.
The `PackitAPIWithUpstreamMixin` "depends" on the `LocalProjectMixin` so it is too late to specify the suffix for the upstream in  `PackitAPIWithUpstreamMixin`.
And I have no other ideas...

I choose `dg` as a name because in the API is called like that, but I can change it in `dist-git` if you think it is better.

Reply:
Yes, I think `dist-git` is better, `dg` makes sense in the code as it is shorter, but I think we can afford to be a bit more explicit in directory names :slightly_smiling_face: